yarn-error.log

```diff
@@ -0,0 +1,8819 @@
+Arguments: 
```

Comment:
Thinking we don't want to be adding the `yarn-error.log` to the repo... Maybe add it to the gitignore?